Automattic/jetpack
# VideoPress: Add Functionality to the VideoChapters Block Prompt

Part of: #27206

#### Changes proposed in this Pull Request:
When chapters are present in a VideoPress block, a message is displayed prompting the user to add a VideoChapters block. This PR adds functionality to the prompt by adding and keeping track of the new VideoChapters block. Since the VideoChapters block is not ready yet, a Text block is currently used as a placeholder.

The changes will affect the following:
* Jetpack
* VideoPress

#### Other information:


#### Jetpack product discussion


#### Does this pull request change what data or activity we track or use?
* No

#### Testing instructions:
1. Ensure that [beta extensions](https://github.com/Automattic/jetpack/tree/trunk/projects/plugins/jetpack/extensions#beta-extensions) are enabled.
2. Modify the description of a VideoPress block to include chapters in the following format:
```00:00 description
01:35 description
04:57 description
12:10 description
```
3. Save the post/page.
4. You should see a prompt to add a VideoChapters block similar to the following image (styling is being updated):
![videochapters-prompt](https://user-images.githubusercontent.com/56307/199624816-05de9019-e917-4684-83ae-331636251780.png)
5. Click the button in the prompt.
6. You should see a new Text block appear below the selected VideoPress block.
7. Select the VideoPress block. You should no longer see the prompt to add the VideoChapters block.
9. Delete the added Text block.
10. Select the VideoPress block. The prompt to add the VideoChapters block should appear.
